README.md

```diff
@@ -3,7 +3,7 @@
 <img align="right" width="180px" src="./logo/kwok.svg"/>
 
 [kwok](https://sigs.k8s.io/kwok) is a toolkit that enables setting up a cluster of thousands of Nodes in seconds.
-Under the scene, all Nodes are simulated to behave like real ones, so the overall approach employes
+Under the scene, all Nodes are simulated to behave like real ones, so the overall approach employs
```

Comment:
https://github.com/kubernetes-sigs/kwok/blob/3ed5d5c2a39795cc6fef2e24b5cbc64df568ea97/site/content/en/_index.md?plain=1#L11

Do the same on site.

Reply:
Done.